src-vue/src/views/SettingsView.vue

```diff
@@ -208,7 +343,6 @@ h3:first-of-type {
     margin-left: 50px;
 }
 
-
```

Comment:
Randomly removed line :>

Reply:
slipped in while trying to beat CSS into vue.
Will revert